infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -250,60 +218,60 @@ private void syncTailSegment(long address) {
 
     @Override
     public void prefixTrim(long address) {
-        if (address < startingAddress) {
+        if (isTrimmed(address)) {
```

Comment:
isTrimmed doesn't carry its weight. Maybe inline it?

Reply:
prefer to follow IDEA and PMD recommendations https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#ifstmtsmustusebraces